assets/src/edit-story/components/canvas/layout.js

```diff
@@ -42,7 +42,7 @@ import useCanvas from './useCanvas';
  */
 
 export const Z_INDEX = {
-  NAV: 1,
+  NAV: 0,
```

Comment:
@dvoytenko Pinging you here in case you see in issues with changing this to 0.

Reply:
Hmm. The reason it was set to `1` is to definitely show it on top of the selection. Also what happens here with the frame itself? Would the frame block actions too with the new z-index? Or is it just the selection?

Reply:
I asked for confirmation on the issue itself, but let's first confirm the current behavior. We basically have two elements here: an element's frame and a moveable control box. Both could be blocking the page menu if we change the z-index. I think the control box (selection) is ok either way and up to the product decision, but a frame should never block the page menu because it's not even visible unless you hover on it.

Reply:
Looks like the behavior was confirmed.

Hmm, so the frame should be not allowed to overflow but the selection yes. The z-index would still need a change then I suppose, and the frame needs hiding though somehow.